native/cocos/scene/Pass.h

```diff
@@ -342,7 +342,7 @@ class Pass : public RefCounted {
 
     const IProgramInfo *_shaderInfo; // weakref to template of ProgramLib
     MacroRecord _defines;
-    Record<ccstd::string, IPropertyInfo> _properties;
+    PassPropertyInfoMap _properties;
```

Comment:
Please add comment to describe why use `PassPropertyInfoMap `.

Reply:
The definitions of the types `Record<ccstd::string, IPropertyInfo>` and `PassPropertyInfoMap` (`ccstd::unordered_map<ccstd::string, IPropertyInfo>`) were exactly the same, so the same literal type can be used directly to simplify the expression.